pkg/pronom/pronom_test.go

```diff
@@ -2,17 +2,139 @@ package pronom
 
 import (
 	"path/filepath"
+	"reflect"
+	"sort"
 	"testing"
 
 	"github.com/richardlehane/siegfried/pkg/config"
 )
 
 var p *pronom
 
+var DataPath string = filepath.Join("..", "..", "cmd", "roy", "data")
+
 func TestNew(t *testing.T) {
-	config.SetHome(filepath.Join("..", "..", "cmd", "roy", "data"))
+	config.SetHome(DataPath)
 	_, err := NewPronom()
 	if err != nil {
 		t.Error(err)
 	}
 }
+
+// setMinimalParams configures our tests to use the minimal fixtures in
+// the test data directory.
+func setMinimalParams() {
+	config.SetDroid("DROID_minimal.xml")
+	config.SetPRONOMReportsDir("pronom_minimal")
```

Comment:
Call like config.SetLimit("fmt/1", "fmt/11", "fmt/14", "fmt/3", fmt/5")() to set the limit immediately (needed if you are using NewPronom() rather than pronom.New(opts...)

Reply:
If I understand correctly, rather than looking at the new files as dependencies per se, the intention here was to make the tests less sensitive to changes in the signature file, and only sensitive to changes in the code. So there's a bit of overhead with this PR to achieve that, but in future, DROID_SignatureFile_v109.xml can be upgraded for the tests that use it, without having to fixup and breakages in these newly introduced tests, e.g. if more format classifications are added. I believe it can lead to more flexibility in how test cases are added in future. 

What do you think? I can revert back to the current approach if it doesn't work? Another way to do this, to avoid creating more fixtures here, is to see the concept used for creating the skeleton files in this PR through to the end and maybe output the signature files to a temporary location on the filesystem too?

Reply:
That's a good point but there probably won't be too much churn in the tests: you chose pretty stable formats, none of them has been updated since 2013! If there are occasional failures due to PRONOM updates it might also be interesting to know?